experimental/devices/ccs811/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package CCS811 controls CCS811 Volatile Organic Compounds sensor via
+// I²C interface.
+//
+// Datasheet
+//
+// https://ams.com/ccs811
+
```

Comment:
This won't show as package documentation unless you remove this line. See https://blog.golang.org/godoc-documenting-go-code for more background.